contrib/log_correlation/stackdriver/src/test/java/io/opencensus/contrib/logcorrelation/stackdriver/OpenCensusTraceLoggingEnhancerTest.java

```diff
@@ -105,6 +177,86 @@ private static LogEntry getEnhancedLogEntry(String projectId, Span span) {
     }
   }
 
+  private static void assertContainsNoTracingData(LogEntry logEntry) {
+    assertThat(logEntry.getLabels()).doesNotContainKey("sampled");
+    assertThat(logEntry.getTrace()).isNull();
+    assertThat(logEntry.getSpanId()).isNull();
+  }
+
+  @Test
+  public void spanSelectionDefaultIsSampledSpans() {
+    assertThat(new OpenCensusTraceLoggingEnhancer().getSpanSelection())
+        .isEqualTo(SpanSelection.SAMPLED_SPANS);
+  }
+
+  @Test
+  @SuppressWarnings("TruthConstantAsserts")
+  public void spanSelectionPropertyName() {
+    assertThat(OpenCensusTraceLoggingEnhancer.SPAN_SELECTION_PROPERTY_NAME)
+        .isEqualTo(OpenCensusTraceLoggingEnhancer.class.getName() + ".spanSelection");
+  }
+
+  @Test
+  public void setSpanSelectionWithSystemProperty() {
+    try {
+      System.setProperty(OpenCensusTraceLoggingEnhancer.SPAN_SELECTION_PROPERTY_NAME, "NO_SPANS");
+      assertThat(new OpenCensusTraceLoggingEnhancer().getSpanSelection())
+          .isEqualTo(SpanSelection.NO_SPANS);
+    } finally {
+      System.clearProperty(OpenCensusTraceLoggingEnhancer.SPAN_SELECTION_PROPERTY_NAME);
+    }
+  }
+
+  @Test
+  public void setSpanSelectionWithLoggingProperty() throws IOException {
+    try {
+      LogManager.getLogManager()
+          .readConfiguration(
+              CharSource.wrap(
+                      OpenCensusTraceLoggingEnhancer.SPAN_SELECTION_PROPERTY_NAME + "=ALL_SPANS")
```

Comment:
Please change this to NO_SPANS if you change the default. :)

Reply:
Do you mean change it to something other that the default?  I changed it to SAMPLED_SPANS when I changed the default.

Reply:
Yep, that's what I was after. SAMPLED_SPANS is fine.